Simplenote/Classes/SPTagEntryField.m

```diff
@@ -22,7 +22,7 @@ - (instancetype)init
         self.textColor = [UIColor simplenoteTagViewTextColor];
         self.placeholdTextColor = [UIColor simplenoteTagViewPlaceholderColor];
         self.textAlignment = NSTextAlignmentNatural;
-        self.placeholder = NSLocalizedString(@"Tag...", @"Placeholder text in textfield when adding a new tag to a note");
+        self.placeholder = NSLocalizedString(@"Tag…", @"Placeholder text in textfield when adding a new tag to a note");
```

Comment:
Alternatively, we could revert back to the text that was in the code originally:

```suggestion
        self.placeholder = NSLocalizedString(@"Add a tag…", @"Placeholder text in textfield when adding a new tag to a note");
```

I updated the code to "Tag" in the PR against the release branch just so we'd keep consistent behavior with the App Store build. The release branch is not the place where to make non-bugfix changes.

My rational for keeping "Tag" would be that if someone updated the `.strings` file they must have wanted the copy to be that. From a UX perspective, I think both "Add a tag…" and "Tag…" get to the point of what tapping that bit of UI would do. But I don't have a strong opinion on this.

@SylvesterWilmott what's your two cents?

Reply:
I enabled auto-merge on this PR. If you are reviewing and reading this and feel that "Add a tag…" would be a better option, feel free to apply the suggestion above and approve 👍

Reply:
Thanks @mokagio 

`Tag...` looks good to me!